### PR TITLE
Feature/213 list of invitations

### PR DESCRIPTION
**Pull request by alejandro-castellon**
[AB#213](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/213)

# Overview

- Implement design for project's list of invitations

- Integration with API

## Screenshots 
Design
MicrosoftTeams-image

Implementation
image

### CODE REVIEW

**Review by ela-ocana-jalafund — APPROVED** [review]

**Review by freddy-perez-jalafund — APPROVED** [review]
Approved

File: src/components/invitations/InvitationsAPI.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const API_NAME = "projects";
+const API_VERSION = "v1";
+
+export async function getInvitationsByProject(id) {
+  try {
+    const { data } = await axios.get(
+      `${API_HOST}/api/${API_VERSION}/${API_NAME}/${id}/invitations`
+    );
+    return data;
+  } catch (error) {
```
Comment (jorge-lema-jalafund) [inline src/components/invitations/InvitationsAPI.js:15]: Improve your error handling here or in the component. I always get the following error when entering a project details. Note that if the project doesn't have any invitations... it isn't an error. Check if the problem is here or from the API
image

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

File: src/components/invitations/InvitationsAPI.js (hunk 1 above)
Comment (alejandro-castellon) [reply src/components/invitations/InvitationsAPI.js:15]: image
In projects details I got 200 as response of the request and doesnt appear the alert error.

**Review by jorge-lema-jalafund — APPROVED** [review]
approved!

**Pull request merged**